.eslintrc.js

```diff
@@ -26,7 +26,7 @@ module.exports = {
     },
   ],
   parserOptions: {
-    ecmaVersion: 2018,
+    ecmaVersion: 2020,
```

Comment:
https://eslint.org/blog/2020/07/eslint-v7.5.0-released#optional-chaining-support

Reply:
Note that we have to be somewhat more careful about this on the website than in the add-on, but [it looks like](https://caniuse.com/mdn-javascript_operators_optional_chaining) only old versions of Safari and Chrome accounting for <3% of global usage don't support optional chaining, so I guess that's fine? Might not hold for everything in ES2020 though.